challenges/day-02/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Day 02</title>
+    <style>
+        h1 {
+            width: 20%;
+            height: 20%;
+            margin: 20px;
+            padding: 5px 5px 5px 15px;
+            background-color: rgba(255, 255, 255, 0.2);
+            border-color: white;
+            color: white;
```

Comment:
This is a super minor nitpick, and more informational than anything else. Color theory is a huge topic and something that I don't think I'll ever fully understand, but one concept/idea I come across often is to avoid using pure white or black in web design. Has something to do with contrast and helps reduce the harshness that pure white can sometimes cause on our eyes. It's always better to use a shade instead. This is not a hard requirement, and using white here is perfectly acceptable.

In my case I used the following value: `#ddd`. That ends up being a shade of white'ish grey. The above value is a hexadecimal value often used for colors. There is a recent trend to move to HSL instead of hex, but I recommend learning hex first as it is still the standard pretty much everywhere. 

Hexadecimal goes from 0-F (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, A, B, C, D, E, F). The way hex colors work is you assign a value for each of the RGB (Red, Green, Blue) values. So if you want a perfect red, that would be `#F00`. If you want Purple, you would do `#F0F`, since purple is just red and blue combined. RGB is additive so if you want white, you essentially turn on all colors `#FFF`, and for black you turn off all colors `#000`. Lastly, sometimes instead of 3 digits, you may see 6 digits (ie #F6C509). That just gives you more granularity in your color choices but it essentially works the same (#RRGGBB), so #FFFFFF is equivalent to #FFF. 

Hope that helps, we can chat more about it if you have questions, just wanted to give a super quick rundown of how colors work.

Reply:
Ok I tried it out and I see. I was learning about this in one of the courses however, it did not explain it as well. How would I go about doing that and then adding the transparency to it though? I was confused about that part and had to google it when I first did it and that was the way I found. I will try to find out first but if you can run that by me when we next chat it would be super helpful.

Reply:
I was only referring to the `color` property here. Border color is not necessary and the background was perfectly fine since you were using an opacity (or alpha) value already.

And yeah, we can totally chat about this in more detail if you'd like.

Reply:
Ah ok. I went ahead and got rid of the border-color and it still looks the same. Got it. I would like to chat more about it so I can understand it better. Also, I did try to change the format for the background-color so that it is in the same hexadecimal format.